### PR TITLE
<MarketingLayout/> - adding direction prop

### PR DESCRIPTION
**Pull request by TalZa1810**
### 🔦 Summary
Adding direction prop to `<MarketingLayout/>`

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [x] Story
  - [x] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [x] Visual test

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6691

File: packages/wix-style-react/src/MarketingLayout/MarketingLayout.js [hunk 1]
```diff
@@ -137,25 +140,38 @@ class MarketingLayout extends React.PureComponent {
...
-      <Layout gap="24px">
-        {inverted ? [imageCell, contentCell] : [contentCell, imageCell]}
-      </Layout>
-    ) : (
-      <Layout>
-        {inverted
-          ? [imageCell, contentCell, spacerCell]
-          : [contentCell, spacerCell, imageCell]}
+    const { direction, size } = this.props;
+    const { spacer, image, content } = cellSpansBySize[size];
```
Comment (TalZa1810) [inline packages/wix-style-react/src/MarketingLayout/MarketingLayout.js:144]: **size small:**
- image: 3
- spacer: 1
- content: 8

**Comment by TalZa1810** [comment]
no image placeholder for vertical direction.

**Comment by TalZa1810** [comment]
@milkyfruit 
I'm removing the `large` example since we want to deprecate this.

**Review by milkyfruit — APPROVED** [review]
@TalZa1810 looks good!

**Comment by TalZa1810** [comment]
fails on a visual which is not related.

**Comment by TalZa1810** [comment]
will be available in the next release
FYI @yogevitz

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  Project Name: **wix-ui-tpa**

  You can find it under one of the following URLs:
  * https://bo.wix.com/pages/wix-ui-tpa.pr_6691/

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  Project Name: **wix-style-react**

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6691

**Pull request merged**